src/responder/ifp/ifpsrv.c

```diff
@@ -341,7 +341,7 @@ int main(int argc, const char *argv[])
 
     sss_set_logger(opt_logger);
 
-    ret = server_setup("sssd[ifp]", 0, 0, 0,
+    ret = server_setup("ifp", 0, 0, 0,
```

Comment:
If square brackets are not requested by some log processing tools? We are free to fully get rid of them?

Reply:
I am not aware of. Please let me know if there are any.